pkg/commands/compute/pack.go

```diff
@@ -48,13 +50,12 @@ func (c *PackCommand) Exec(_ io.Reader, out io.Writer) (err error) {
 	if err = c.manifest.File.ReadError(); err != nil {
 		return err
 	}
-	name := sanitize.BaseName(c.manifest.File.Name)
-	pkg := fmt.Sprintf("pkg/%s/bin/main.wasm", name)
-	dir := filepath.Dir(pkg)
-	err = filesystem.MakeDirectoryIfNotExists(dir)
+	bin := "pkg/package/bin/main.wasm"
+	bindir := filepath.Dir(bin)
```

Comment:
ℹ️ I'm just renaming variables for greater clarity, as the previous names were a bit too ambiguous.